src/models/chan.js

```diff
@@ -9,6 +10,7 @@ Chan.Type = {
 };
 
 var id = 0;
+var config = Helper.getConfig();
```

Comment:
This breaks the `--home` command line switch as the config gets loaded when the models are `require()`'ed so before the command line switches can be processed.